specification/ai/HealthInsights/HealthInsights.RadiologyInsights/model.radiologyinsights.tsp

```diff
@@ -57,17 +58,28 @@ model FindingOptions {
   provideFocusedSentenceEvidence?: boolean;
 }
 
-@resource("radiology-insights/jobs")
 @doc("Response for the Radiology Insights request.")
-model RadiologyInsightsResult
-  is HealthInsightsOperationStatus<RadiologyInsightsInferenceResult>;
+@resource("radiology-insights/jobs")
+model RadiologyInsightsJob {
+  @doc("The request data for the operation.")
+  @visibility("read", "create")
+  jobData?: RadiologyInsightsData;
```

Comment:
I think `jobData` should be required.
```suggestion
  jobData: RadiologyInsightsData;
```

Reply:
In our review with Jeffery, we aggreed this will not be required as it contains in the Get schema.
Jeffery suggested, this will be optional,  vlidated by the service, and should return if expand=jobData.